axelor-base/src/main/resources/i18n/messages_fr.csv

```diff
@@ -154,14 +154,23 @@
 "Add Guests","Ajouter des invités",,
 "Additionnal","Ajout",,
 "Address","Adresse",,
+"Address Field is missing : %s",,,
+"Address L2 template",,,
+"Address L3 template",,,
+"Address L4 template",,,
+"Address L5 template",,,
+"Address L6 template",,,

```

Comment:
Change these titles for "Address line 2 template", "Address line 3 template", etc.